src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1386,6 +1386,7 @@
   "com.minecolonies.coremod.research.requirementnotmet": "Minimum research requirement not met!",
   "com.minecolonies.coremod.research.started": "Research successfully started!",
   "com.minecolonies.coremod.research.havetounlock": "Build a University and research this hut to unlock it!",
+  "com.minecolonies.coremod.research.crafted": "You just crafted the %s, to build it you need research it at the university first.",
```

Comment:
```suggestion
  "com.minecolonies.coremod.research.crafted": "You just crafted the %s. To build it, you must research it at the University.",
```

Reply:
you could understand that as "you have to research everything before you can build this", as "complete the research" can mean "finish researching this one thing" or "research everything".

Reply:
I replaced 'complete the research' with 'complete its research.' Does that make more sense?

Reply:
yes, this way i can't mean "research everything" anymore.